extras/docker/fromsource/heketi-start.sh

```diff
@@ -34,8 +34,18 @@ if [[ -d "${HEKETI_PATH}" ]]; then
     stat "${HEKETI_PATH}/heketi.db" | tee -a "${HEKETI_PATH}/container.log"
     # Workaround for scenario where a lock on the heketi.db has not been
     # released.
-    flock -w 60 "${HEKETI_PATH}/heketi.db" true
-    if [[ $? -ne 0 ]]; then
+    # This code uses a non-blocking flock in a loop rather than a blocking
+    # lock with timeout due to issues with current gluster and flock
+    # ( see rhbz#1613260 )
```

Comment:
does this mean we are working around a (new?) bug in gluster here?
That's fine, but just want to make sure that this gets tracked as well....

Reply:
Yeah, we'll make sure a bug remains open for the gluster side of the issue.